src/vcf/validate_optional_policy.cpp

```diff
@@ -146,6 +152,36 @@ namespace ebi
         }
     }
 
+    void ValidateOptionalPolicy::check_body_entry_alt_gvcf_end(ParsingState & state, Record const & record) const
+    {
+        if (std::find(record.alternate_alleles.begin(), record.alternate_alleles.end(), GVCF) != record.alternate_alleles.end()
```

Comment:
I'm so tired of this `!= end()` idiom that I'm considering doing a generic function like this:
```
bool contains(collection, element) {
    return std::find(collection.begin(), collection.end(), element) != collection.end();
}
```
@cyenyxe thoughts?

Reply:
not done in next commit

Reply:
And what about this @jmmut @cyenyxe?
Should this be done in a separate PR or in this one itself?

Reply:
Test commit added for this: https://github.com/EBIvariation/vcf-validator/pull/91/commits/c426c36c930cd84e2c7803491a9315ede5de29bf.
Will remove it if not needed.